consentconstants/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/prebid/go-gdpr/consentconstants
```

Comment:
Why not define a single go module at the root?

Reply:
@SyntaxNode 

Fixed.

No good reason. I was thinking that breaking them up into more modules would make it easier for management when tcf v2 gets implemented, but there are cleaner ways to handle that when it comes.